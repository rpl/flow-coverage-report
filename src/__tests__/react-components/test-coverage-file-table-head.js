'use babel';

import React from 'react';
import renderer from 'react-test-renderer';

import {BASE_DIR} from './common';

const REACT_COMPONENT = `${BASE_DIR}/coverage-file-table-head`;

test('<FlowCoverageFileTableHead />', () => {
  const FlowCoverageFileTableHead = require(REACT_COMPONENT).default;
  const tree = renderer.create(<FlowCoverageFileTableHead/>).toJSON();
  expect(tree).toMatchSnapshot();
});

test.todo('<FlowCoverageFileTableHead /> with missing props');
