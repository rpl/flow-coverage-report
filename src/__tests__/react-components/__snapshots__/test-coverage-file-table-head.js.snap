// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FlowCoverageFileTableHead /> 1`] = `
<thead>
  <tr>
    <th
      data-sort-default={true}
    >
      Filename
    </th>
    <th>
      Annotation
    </th>
    <th>
      Percent
    </th>
    <th>
      Total
    </th>
    <th>
      Covered
    </th>
    <th>
      Uncovered
    </th>
  </tr>
</thead>
`;
